libraries/debug-certificate-manager/src/CertificateManager.ts

```diff
@@ -414,10 +425,14 @@ export class CertificateManager {
     certificate.setSubject(subjectAttrs);
     certificate.setIssuer(issuerAttrs);
 
-    const subjectAltNames: readonly IAltName[] = subjectNames.map((subjectName) => ({
+    const subjectAltNames: IAltName[] = subjectNames.map((subjectName) => ({
       type: 2, // DNS
       value: subjectName
     }));
+    subjectAltNames.push({
```

Comment:
So we're always adding `127.0.0.1`? The idea being that this is only ever going to be used for local development?

Reply:
I'm open to adding another parameter for IP Addresses to authenticate, though since it's a self-signed cert it should probably only be used for local development.